app/services/chat_channels/create.rb

```diff
@@ -0,0 +1,16 @@
+module ChatChannels
```

Comment:
Sorry I didn't catch this in the initial reviews but this service doesn't seem to be useful (it wasn't in the previous incarnation as well). We should directly call `user.chat_channels.create()` and get rid of this as it's an entire service to replace one line of code